cli/src/commands/file/mod.rs

```diff
@@ -24,6 +25,8 @@ use crate::ui::Ui;
 pub enum FileCommand {
     Print(print::PrintArgs),
     Chmod(chmod::ChmodArgs),
+    #[command(visible_alias("ls"))]
```

Comment:
We need to come to some sort of consensus what core aliases are, see #3218 and https://github.com/martinvonz/jj/pull/3242#pullrequestreview-1923133220. cc @martinvonz @thoughtpolice

Reply:
Ok, I will simply remove the alias for now